src/index.ts

```diff
@@ -167,6 +169,19 @@ export class UltravoxSession {
     this.sendData({ type: 'input_text_message', text });
   }
 
+  get isCallerMuted(): boolean {
+    return this._isCallerMuted;
+  }
+
+  set isCallerMuted(newMuteStatus: boolean) {
+    if (!this.room?.localParticipant) {
+      throw new Error('Cannot set isCallerMuted.');
```

Comment:
Why not update the other place we setMicrophoneEnabled to use our new private state and then just skip calling setMicrophoneEnabled in this case?